src/main/java/core/basesyntax/converter/DataConverterImpl.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.converter;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+    @Override
+    public List<FruitTransaction> covertToTransaction(List<String> data) {
+        List<FruitTransaction> transactions = new ArrayList<>();
+
+        for (int i = 1; i < data.size(); i++) {
```

Comment:
Avoid using magic numbers. It's not clear why the loop starts from 1. If you are skipping the header, define a constant to explain the purpose, such as 'int HEADER_LINE_INDEX = 0;'.